Gemfile.lock

```diff
@@ -528,7 +528,7 @@ GEM
       jwt (~> 2.1, >= 2.1.0)
       rest-client (~> 2.0, >= 2.0.2)
     pusher-signature (0.1.8)
-    raabro (1.3.1)
+    raabro (1.4.0)
```

Comment:
dependency of `fugit` which is already a transitive dependency, nothing to report